### ISSUE TITLE
Allow requests in parallel.

### ISSUE THREAD

**Issue by Levino** [figo-connect/node-figo#20]
New ticket to replace messy #15 

Currently one can one do one query at a time for a single "Session" Object. This is not necessary and makes it a pain to use this lib for any node developer. The sdk sould support an arbitrary number of parallel queries.

**Comment by JeremyCraigMartinez** [comment]
resolved in version [1.4.3](https://github.com/figo-connect/node-figo/tree/v1.4.3)
